data-tool/scripts/transfer_cprd_corps.sql

```diff
@@ -1,9 +1,12 @@
-connect cprd_pg;
-
-vset transfer.threads=8
+vset cli.settings.ignore_errors=false
+vset cli.settings.transfer_threads=8
 vset format.date=YYYY-MM-dd'T'hh:mm:ss'Z'
 vset format.timestamp=YYYY-MM-dd'T'hh:mm:ss'Z'
 
+connect cprd_pg;
+
+learn schema public;
+
```

Comment:
started updating extract to work with latest dbschema cli.  

there seems to be new issues with extract script.

will address in near future but not in the work for this ticket.

Reply:
what's the version of tool you are using now?

Reply:
i'm on 9.6.2